packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -1,9 +1,10 @@
 name: image_picker
 description: Flutter plugin for selecting images from the Android and iOS image
   library, and taking new pictures with the camera.
+homepage: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker
```

Comment:
Please don't add this; it will just result in an extra link on pub.dev that is misleading (since it's not actually a homepage that's distinct from the repository.

Reply:
Woops, I think this got in by merging with an older version of master, my bad. Fixed.